ivy/functional/frontends/torch/spectral_ops.py

```diff
@@ -0,0 +1,15 @@
+import ivy
+from ivy.functional.frontends.torch.func_wrapper import (
+    to_ivy_arrays_and_back,
+)
+from ivy.func_wrapper import with_supported_dtypes
+
+
+@with_supported_dtypes({"2.0.1 and below": "int"}, "torch")
```

Comment:
the value for the dict should be a tuple of supported dtypes. Ex: `("int32", "int64")` (You should check to see exactly which int dtypes are supported)

Reply:
That's right, I missed this...let me try it..